client/src/components/flight-details.component.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import { getFlightDetails } from "../utils/skyscanner";
+import FlightDetailsByDirection from "../components/flight-details-by-direction.component";
+
+function FlightDetails(props) {
+  const useStyles = makeStyles({
+    detailsContainer: {
+      textAlign: "center",
+      marginTop: "35px",
+      border: "1px solid #D3D3D3",
+      borderRadius: "5px",
+    },
+  });
+  const classes = useStyles();
+
+  const { quoteDetails, places, carriers } = props;
+  //THIS DEFINTELY NEEDS A UTIL FUNCTION OF ITS OWN LATER
```

Comment:
Can't we combine this into getFlightDetails and just have it return two objects: `outgoingDetails` and `inboundDetails`?

Reply:
Yes, just combined it! The repetitive part is now handled within the getFlightDetails function.

Reply:
Actually, getFlightDetails was repetitive to being with so not much changed on that part